css/index.css

```diff
@@ -1,3 +1,333 @@
 /* Use your own code or past solution for Great Idea Web Page CSS here! */
 
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
 
+html,
+body,
+div,
+span,
+applet,
+object,
+iframe,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre,
+a,
+abbr,
+acronym,
+address,
+big,
+cite,
+code,
+del,
+dfn,
+em,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  font: inherit;
+  vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
+}
+body {
+  line-height: 1;
+}
+ol,
+ul {
+  list-style: none;
+}
+blockquote,
+q {
+  quotes: none;
+}
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
+  content: "";
+  content: none;
+}
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+  box-sizing: border-box;
+}
+
+html,
+body {
+  height: 100%;
+  font-family: "Titillium Web", sans-serif;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5 {
+  font-family: "Bangers", cursive;
+  letter-spacing: 1px;
+  margin-bottom: 15px;
+}
+
+/* Your code starts here! */
+.container {
+  margin: 0px auto;
+  width: 50%;
+}
+
+header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  margin-top: 30px;
+  margin-bottom: 30px;
+}
+.nav-bar a {
+  margin-right: 50px;
+  text-decoration: none;
+  color: gray;
+}
+.logo {
+  flex-direction: flex-end;
+}
+article {
+  display: flex;
+  flex-direction: row;
+  align-content: center;
+  margin-top: 80px;
+  border-bottom: 1px solid black;
+}
+.intro-title {
+  flex-wrap: wrap;
+  padding-top: 45px;
+  padding-left: 20px;
+  width: 256px;
+  height: 300px;
+  font-size: 64px;
+  text-align: center;
+  margin-top: 30px;
+}
+.btn {
+  display: flex;
+  width: 250px;
+  justify-content: center;
+  margin-bottom: 50px;
+  padding-left: 20px;
+}
+h2 {
+  border: 1px solid black;
+  width: 150px;
+  height: 30px;
+  text-align: center;
+  padding-top: 5px;
+}
+.second-img {
+  display: flex;
+  width: 800px;
+  justify-content: flex-end;
+  margin-bottom: 50px;
+}
+.section-one {
+  display: flex;
+  flex-direction: row;
+  margin: 50px 0px;
+}
+.middle-img {
+  width: 900px;
```

Comment:
This image is giving you troubles. Now that you can use percentages on images they will be a little easier to deal with.